app/services/trainees/create_timeline_events.rb

```diff
@@ -91,12 +91,20 @@ def single_event_title
     end
 
     def state_change_title
+      I18n.t("components.timeline.titles.trainee.#{state_change_action}")
+    end
+
+    def state_change_action
       change_from, change_to = audited_changes["state"].map { |change| Trainee.states.key(change) }
 
       if change_from == "deferred" && change_to != "withdrawn"
-        I18n.t("components.timeline.titles.trainee.reinstated")
+        "reinstated"
+      elsif change_to == "recommended_for_award"
+        "recommended_for_#{auditable.award_type.downcase}"
+      elsif change_to == "awarded"
```

Comment:
This can probably be another PR but for other enum fields we're centralising the values via config constants so we can re-use the constants instead of the literal string values. I wonder if we can do that for the trainee `state` field too

Reply:
agreed! we're using the states quite a bit now, and will continue to do so i think. do you want to add a card to track?